src/sonic-yang-models/tests/files/sample_config_db.json

```diff
@@ -907,6 +907,56 @@
         "PREFIX": {
             "prefix1|1|10.0.0.0/8|8..16": {
             }
+        },
+        "FEATURE": {
+            "bgp": {
+                "auto_restart": "enabled",
+                "has_global_scope": "False",
+                "has_per_asic_scope": "True",
+                "has_timer": "False",
+                "high_mem_alert": "disabled",
+                "state": "enabled"
+            },
+            "database": {
+                "auto_restart": "always_enabled",
+                "has_global_scope": "True",
+                "has_per_asic_scope": "True",
+                "has_timer": "False",
+                "high_mem_alert": "disabled",
+                "state": "always_enabled"
+            },
+            "snmp": {
+                "auto_restart": "enabled",
+                "has_global_scope": "True",
+                "has_per_asic_scope": "False",
+                "has_timer": "True",
+                "high_mem_alert": "disabled",
+                "state": "enabled"
+            },
+            "swss": {
+                "auto_restart": "enabled",
+                "has_global_scope": "False",
+                "has_per_asic_scope": "True",
+                "has_timer": "False",
+                "high_mem_alert": "disabled",
+                "state": "enabled"
+            },
+            "syncd": {
+                "auto_restart": "enabled",
+                "has_global_scope": "False",
+                "has_per_asic_scope": "True",
+                "has_timer": "False",
+                "high_mem_alert": "disabled",
+                "state": "enabled"
```

Comment:
check for telemetry https://github.com/Azure/sonic-buildimage/blob/9d18a35e35d3f57c00ba14cae87568a7a203d5d6/src/sonic-config-engine/minigraph.py#L1549

Reply:
The Telemetry feature also has the same state as other feature
https://github.com/Azure/sonic-buildimage/blob/b3b6938fda9244607fb00bfd36a74bccab0c38a9/files/build_templates/init_cfg.json.j2#L39